common/lib/api-client/cache.js

```diff
@@ -0,0 +1,39 @@
+const LRU = require('lru-cache')
+
+const redisStore = require('../../../config/redis-store')
+
+const lruCache = new LRU()
+
+async function get(key, useRedisCache) {
+  let data
+
+  if (useRedisCache) {
+    data = await redisStore().client.getAsync(key)
+  } else {
+    data = lruCache.get(key)
+  }
+
+  if (typeof data === 'string') {
+    data = JSON.parse(data)
+  }
+
+  return data
+}
+
+async function set(key, response = {}, expiry, useRedisCache) {
+  const { data } = response
```

Comment:
Any reason to pass in the whole response object?

This look like the `set` will return this format:

```json
{
  "data": {
    "foo": "bar"
  }
}
```

but `get` will return:

```json
{
  "foo": "bar"
}
```

Might be more consistent to pass the data expected to be set directly into `set` and then that structure will be exactly what's return from `get`.

Reply:
👍